cw-orch/src/interchain/follow_ibc_execution.rs

```diff
@@ -59,9 +58,67 @@ pub enum AckResponse {
 
     // 3. Then we look for the acknowledgment packet that should always be traced back during this transaction for all packets
 
+
+/// Follow all IBC packets included in a transaction (recursively).
+/// ## Example
+/// ```no_run,ignore
```

Comment:
one of the two does the job. no_run checks compiles while ignore will not do anything.